### ISSUE TITLE
bug when parsing 'static<T>'

### ISSUE THREAD

**Issue by SvyatoslavRazmyslov** [INRIA/spoon#2847]
Bad code:

`public static<T> T identity(T arg) { return arg; }; `

Stack:

```
Error: Position of CtExtendedModifiers: [static] not found in
spoon.SpoonException: Position of CtExtendedModifiers: [static] not found in
    at spoon.support.compiler.jdt.PositionBuilder.setModifiersPosition(PositionBuilder.java:543)
    at spoon.support.compiler.jdt.PositionBuilder.buildPositionCtElement(PositionBuilder.java:328)
    at spoon.support.compiler.jdt.ContextBuilder.enter(ContextBuilder.java:100)
    at spoon.support.compiler.jdt.JDTTreeBuilder.visit(JDTTreeBuilder.java:1059)
    ....
```
Good, if you add a space after 'static':

`public static <T> T identity(T arg) { return arg; }; `

Spoon from master.

**Comment by monperrus** [comment]
closed by #2848

**Comment by monperrus** [comment]
Thanks for the bug report
